notes/README.md

```diff
@@ -0,0 +1,592 @@
+
+| Property | Required | Notes
+| -- | -- | --
+| state | Maybe | If the current state is pending or then this MUST have a value of pending or not be provided. If the current state is expired or then this MUST have a value of pending. If the current state is anything but expired or pending then this request MUST be rejected.
+| receiveAmount | No | assetCode and assetScale MUST match those of the receiver. Not permitted if the receiver is an incoming payment resource or if a sendAmount is also provided.
+| sendAmount | No | assetCode and assetScale MUST match those of the underlying Account. Not permitted if the receiver is an incoming payment resource or if a receiveAmount is also provided.
+
+
+If any other properties are provided the request MUST be rejected.
+
+#### 3.2.5 Authorize Payment
+
+**Method**: PUT
+**Content-Type**: application/op-outgoing-payment-v1+json
+**Accept**: application/op-outgoing-payment-v1+json
+
+If the state is **Pending** then the state can be updated to **Authorized** by a client that has the appropriate permissions.
+
+This will cause the ASPSP to begin executing the payment.
+
+The request body contains an Outgoing Payment resource encoded as JSON with only the following properties: 
+
+| Property | Required | Notes
+| -- | -- | --
+| state | Yes | This MUST have a value of authorized. If the current state is anything but pending then this request MUST be rejected.
+
+If any other properties are provided the request MUST be rejected.
+
+## Types
+
+The following common data types are used in the resource definitions above
+
+### Amount
+
+| Property | Type | Description |
+| -- | -- | --
+| amount | uint64 | The amount that must be paid into the receiver’s account. (Optional)
+| assetCode | string | The asset code of the amount. This SHOULD be an ISO4217 currency code.
+| assetScale | uint32 | The scale of the amount.
+
+## Grants
+
+### Types
+
+The grant types map directly to resource types and include:
+* account
+* incoming-payment
+* outgoing-payment
+
+A grant with the type "account" permits a client to perform actions against the account resources identified by the location property of the grant.
+
+A grant with the type `incoming-payment` permits the client to perform the specified actions on incoming payment resources that are sub-resources of the account resources identified by the location property of the grant.
+
+A grant with the type `outgoing-payment` permits the client to perform the specified actions on outgoing-payment resources that are sub-resources of the account resources identified by the location property of the grant.
+
+### Actions
+
+Actions specify the actions a client can perform against a resource. Different actions are available for different grant types.
+
+*__NOTE:__ We need to validate the account related actions with use cases before we can define the access control properly.*
+
+Incoming Payment
+* Create
+* Read
+* RequestReceipts
```

Comment:
```suggestion
```

Reply:
@wilsonianb - I'm struggling to keep this all up to date here. Can we discuss on HackMD until things stabilise a bit?

https://hackmd.io/@openpayments2

Reply:
:+1: I'll close this for now in favor of that.